.editorConfig

```diff
@@ -0,0 +1,30 @@
+# EditorConfig file: http://EditorConfig.org
+
+# Top-most EditorConfig file.
+root = true
+
+# Unix-style newlines with a newline ending every file.
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = true
+indent_style = space
+
+# 2 space indentation for configuration files.
+[*.{yml,config, src}]
+indent_size = 2
```

Comment:
We actually use erlang-style indentation for all config files and .src files (like .app.src) and usually have everything set to 4 spaces.